tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -124,6 +124,10 @@ class RLoopManager : public RNodeBase {
    std::shared_ptr<TTree> fTree{nullptr};
    Long64_t fBeginEntry{0};
    Long64_t fEndEntry{std::numeric_limits<Long64_t>::max()};
+
+   std::vector<ROOT::RDF::Experimental::RGroupMetaData> fGroupMetaDataVec;
+   std::unordered_map<std::string, ROOT::RDF::Experimental::RGroupMetaData *> fGroupMetaDataMap;
```

Comment:
please add a comment to specify what's the key in this map and that `fGroupMetaDataMap` points into `fGroupMetaDataVec`